sdk/textanalytics/ai-text-analytics/test/textAnalyticsClient.spec.ts

```diff
@@ -105,11 +108,112 @@ describe("[AAD] TextAnalyticsClient", function() {
     });
+
+    it("client gets positive mined opinions", async () => {
+      const documents = [
+        {
+          text: "It has a sleek premium aluminum design that makes it beautiful to look at.",
+          id: "0",
+          language: "en"
+        }
+      ];
+      const results: AnalyzeSentimentResultArray = await client.analyzeSentiment(documents, {
+        mineOpinions: true
+      });
+      assert.equal(results.length, 1);
+      assertAllSuccess(results);
+      const documentSentiment: AnalyzeSentimentSuccessResult = results[0] as AnalyzeSentimentSuccessResult;
+      documentSentiment.sentences.map((sentence) =>
+        sentence.minedOpinions?.map((opinion) => {
+          const aspect = opinion.aspect;
+          assert.equal("design", aspect.text);
+          assert.equal("positive", aspect.sentiment);
+          assert.isAtLeast(aspect.confidenceScores.positive, 0);
+          assert.isUndefined(aspect.confidenceScores.neutral);
+          assert.isAtLeast(aspect.confidenceScores.negative, 0);
+
+          const sleekOpinion = opinion.opinions[0];
+          assert.equal("sleek", sleekOpinion.text);
+          assert.equal("positive", sleekOpinion.sentiment);
+          assert.isAtLeast(sleekOpinion.confidenceScores.positive, 0);
+          assert.isAtLeast(sleekOpinion.confidenceScores.neutral, 0);
+          assert.isAtLeast(sleekOpinion.confidenceScores.positive, 0);
+          assert.isFalse(sleekOpinion.isNegated);
+
+          const premiumOpinion = opinion.opinions[1];
+          assert.equal("premium", premiumOpinion.text);
+          assert.equal("positive", premiumOpinion.sentiment);
+          assert.isAtLeast(premiumOpinion.confidenceScores.positive, 0);
+          assert.isAtLeast(premiumOpinion.confidenceScores.neutral, 0);
+          assert.isAtLeast(premiumOpinion.confidenceScores.positive, 0);
+          assert.isFalse(premiumOpinion.isNegated);
+        })
+      );
+    });
+
+    it("client gets negative mined opinions", async () => {
+      const documents = [
+        {
+          text: "The food and service is not good",
+          id: "0",
+          language: "en"
+        }
+      ];
+      const results: AnalyzeSentimentResultArray = await client.analyzeSentiment(documents, {
+        mineOpinions: true
+      });
+      assert.equal(results.length, 1);
+      assertAllSuccess(results);
+      const documentSentiment: AnalyzeSentimentSuccessResult = results[0] as AnalyzeSentimentSuccessResult;
+      documentSentiment.sentences.map((sentence) => {
+        const foodAspect = sentence.minedOpinions?.[0].aspect;
+        assert.equal("food", foodAspect?.text);
+        assert.equal("negative", foodAspect?.sentiment);
+        assert.isAtLeast(foodAspect?.confidenceScores.positive!, 0);
+        assert.isUndefined(foodAspect?.confidenceScores.neutral);
+        assert.isAtLeast(foodAspect?.confidenceScores.negative!, 0);
+
+        const serviceAspect = sentence.minedOpinions?.[0].aspect;
```

Comment:
this should be `sentence.minedOpinions?.[1].aspect;`

Reply:
done.